### PR TITLE
Label Seeder

### PR DESCRIPTION
**Pull request by Pierre-Voisin**

### CODE REVIEW

**Review by AnneDAMIEN — APPROVED** [review]

**Review by yannduhamel — DISMISSED** [review]

**Review by enerisan — APPROVED** [review]

**Review by angelique888 — DISMISSED** [review]

File: server/database/data/label.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+[
+    {
+        
+        "name": "Végétarien",
+        "image": " ../../../client/src/assets/images/Vegetarien.png",
```
Comment (ioayoub) [inline server/database/data/label.json:5]: dynamic images should be stored in public folder.

**Review by ioayoub — CHANGES_REQUESTED** [review]

File: server/database/data/label.json (hunk 1 above)
Comment (Pierre-Voisin) [reply server/database/data/label.json]: It's done

**Review by yannduhamel — DISMISSED** [review]

File: server/database/data/label.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+[
+    {
+        
+        "name": "Végétarien",
+        "image": " ../../../client/public/images/Vegetarien.png",
```
Comment (ioayoub) [inline server/database/data/label.json:5]: the right syntax is : 
"image": " /images/Vegetarien.png"

**Review by ioayoub — CHANGES_REQUESTED** [review]

**Review by enerisan — DISMISSED** [review]

**Review by angelique888 — DISMISSED** [review]

File: server/database/data/label.json (hunk 2 above)
Comment (Pierre-Voisin) [reply server/database/data/label.json]: Ok syntax changed

**Review by yannduhamel — APPROVED** [review]

**Review by enerisan — APPROVED** [review]

**Review by angelique888 — APPROVED** [review]

**Review by AnneDAMIEN — APPROVED** [review]

**Pull request merged**